contracts/svc/invocable-scripts.yml

```
@@ -0,0 +1,317 @@
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+  schemas:
+    Error:
+      properties:
+        code:
+          description: code is the machine-readable error code.
+          readOnly: true
+          type: string
+          enum:
+            - internal error
+            - not found
+            - conflict
+            - invalid
+            - unprocessable entity
+            - empty value
+            - unavailable
+            - forbidden
+            - too many requests
+            - unauthorized
+            - method not allowed
+            - request too large
+            - unsupported media type
+        message:
+          readOnly: true
+          description: message is a human-readable message.
+          type: string
+        op:
+          readOnly: true
+          description: op describes the logical code operation during error. Useful for debugging.
+          type: string
+        err:
+          readOnly: true
+          description: err is a stack of errors that occurred during processing of the request. Useful for debugging.
+          type: string
+      required:
+        - code
+        - message
+    Script:
+      properties:
+        id:
+          readOnly: true
+          type: string
+        name:
+          type: string
+        description:
+          type: string
+        orgID:
+          type: string
+        script:
+          description: script to be executed
+          type: string
+        language:
+          $ref: '#/components/schemas/ScriptLanguage'
+        url:
+          type: string
+          description: invocation endpoint address
+        createdAt:
+          type: string
+          format: date-time
+          readOnly: true
+        updatedAt:
+          type: string
+          format: date-time
+          readOnly: true
+      required:
+        - name
+        - orgID
+        - script
+    Scripts:
+      type: object
+      properties:
+        scripts:
+          type: array
+          items:
+            $ref: '#/components/schemas/Script'
+    ScriptCreateRequest:
+      type: object
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        orgID:
+          type: string
+        script:
+          description: script to be executed
+          type: string
+        language:
+          $ref: '#/components/schemas/ScriptLanguage'
+      required:
+        - name
+        - orgID
+        - script
+        - language
+    ScriptUpdateRequest:
+      type: object
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        script:
+          description: script is script to be executed
+          type: string
+    ScriptHTTPResponseData:
+      description: The data sent to end user when a script is invoked using http. User defined and dynamic
+      type: object
```

The current implementation returns response with following HTTP headers:

```
header: Date: Tue, 19 Oct 2021 10:22:19 GMT
header: Content-Type: text/plain; charset=utf-8
header: Transfer-Encoding: chunked
header: Connection: keep-alive
header: Vary: Accept-Encoding
header: trace-id: 78d6cc5dffa0b7d6
header: trace-id: 78d6cc5dffa0b7d6
header: trace-sampled: false
header: trace-sampled: false
header: vary: Accept-Encoding
header: Strict-Transport-Security: max-age=15724800; includeSubDomains
header: X-Influxdb-Build: Cloud
,result,table,_start,_stop,_time,_value,_field,_measurement,language,license,name,owner
...
```

Is `string` type more suitable?

```suggestion
      type: string
```